EIPS/eip-5173.md

```diff
@@ -0,0 +1,429 @@
+---
+eip: 5173
+title: NFT Future Rewards (nFR) Standard
```

Comment:
We try to discourage the word "standard" in titles and descriptions, because it tends to end up in every ERC.

```suggestion
title: NFT Future Rewards (nFR)
```

You'll probably also want to take a peek at https://github.com/ethereum/EIPs/runs/7393609255?check_suite_focus=true to see the other problems the linter is reporting.